metadata.rb

```diff
@@ -1430,14 +1430,6 @@
 ##
 ##
 
-attribute "jupyter/user",
-          :description => "Jupyter username to run notebook servers as",
-          :type => 'string'
-
-attribute "jupyter/group",
```

Comment:
Could you also remove the `serving/user` and `serving/group`?

Reply:
doing that. I don't know why I missed it.